src/main/java/org/testng/GuiceHelper.java

```diff
@@ -71,61 +75,61 @@ private static Module getParentModule(ITestContext context) {
     if (!Module.class.isAssignableFrom(parentModule)) {
       throw new TestNGException("Provided class is not a Guice module: " + parentModule.getName());
     }
+    if (!context.getGuiceModules((Class<? extends Module>) parentModule).isEmpty()) {
+      return context.getGuiceModules((Class<? extends Module>) parentModule).get(0);
```

Comment:
what if there is more than 1 element?

Reply:
@juherr - From my understanding of the implementation, I dont think there could be a situation wherein there are more than 1 elements in that list. That is why I specifically kept it as get `0th` element.

Reply:
Could you add an exception in case there is more? It will allow us to follow it.

Reply:
Fixed